test/sharness/t0110-gateway.sh

```diff
@@ -63,6 +65,19 @@ test_expect_success "GET invalid path errors" '
   test_must_fail wget http://127.0.0.1:5002/12345
 '
 
+
+test_expect_success "GET webui resolves" '
+  wget http://127.0.0.1:5002/ipfs/$webui_hash -O expected
+  wget "http://127.0.0.1:5001/webui" -O actual1
+  wget "http://127.0.0.1:5001/webui/" -O actual2
+  test_cmp expected actual1 &&
+  test_cmp expected actual2 &&
```

Comment:
So what this tests:
Is whatever I fetched the same between all 3 wgets

Ideally If the node cannot get to the webui files, it still ought to return the same value for all 3 wgets. This might be worth checking (the easiest way is to test on a network isolated machine, which I cannot do myself)